### ISSUE TITLE
Integrate with AI to auto-generate blazons

### ISSUE THREAD

**Issue by Taliesin1** [Azgaar/Armoria#153]
I have found that both ChatGPT and Perplexity do an astonishingly good job at blazoning arms that I upload to them. I wonder is there's an opportunity to use their APIs to automatically generate blazons for your AWESOME arms generator. Man, I love this thing so much! I wish I'd have had it ten years ago when I way running my King Arthur Pendragon RPG campaign! It is stupendously good! Congrats on such an amazing accomplishment!

**Comment by Azgaar** [comment]
Hello. It's obviously possible, but I'm not sure if it's really required. Blazoning is a very specific part or art, most people don't know about it... As for gpt-4, I've tried it, but I don't think it's super good. We would need a better trained AI if we want to get correct blazons. But it's should be enough for a based blazon generation.

**Comment by Taliesin1** [comment]
Yes, it's not required. My experiments with chatGPT 4o have been surprisingly good. Not perfect or infallible but one could always put a disclaimer that it was generated with generative AI and may not be 100% reliable. But thanks for your consideration. Love this tool!
